test/integration/BarbellTest.cpp

```diff
@@ -0,0 +1,296 @@
+#include "BarbellTest.h"
+
+#include "api/ConferenceEndpoint.h"
```

Comment:
We should check which of these includes are needed. It looks like a few may not be needed.